### PR TITLE
chore: add UpdateRequests reconciliation details

### PR DESCRIPTION
**Pull request by realshuting**
## Related issue #
https://github.com/kyverno/kyverno/pull/9157

## Proposed Changes
This PR adds failure handling logic for UR reconciliations.

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [x] I have inspected the website preview for accuracy.
- [x] I have signed off my issue.

### CODE REVIEW

**Review by MariamFahmy98 — APPROVED** [review]

File: content/en/docs/Writing policies/generate.md [hunk 1]
```diff
@@ -410,6 +410,8 @@ A `UpdateRequest` status can have one of four values:
 * `Pending`: the request is yet to be processed or the resource has not been created
 * `Skip`: marked when triggering the generate policy by adding a label/annotation to the existing resource, while the selector is not defined in the policy itself.
 
+Note that Kyverno retries up to three times to reconcile a `Failed` `UpdateRequest`, and the `UpdateRequest` will be garbage collected if it exceeds the retry threshold.
```
Comment (chipzoller) [inline content/en/docs/Writing policies/generate.md:413]:
```suggestion
Note that Kyverno will retry up to three times to reconcile an `UpdateRequest` in a `Failed` status. The `UpdateRequest` will be garbage collected if it exceeds the retry threshold.
```

**Review by chipzoller — CHANGES_REQUESTED** [review]
Seems like this needs to also capture the new field.

**Comment by realshuting** [comment]
> Seems like this needs to also capture the new field.

It's self-explained in the CR [spec](https://github.com/kyverno/kyverno/pull/9157/files#diff-65ed24ceacc89ddea618d14d404077b07db0820a18e49586f8d78a0c27cce09eR102-R107), do we want to call it out in docs?

**Comment by chipzoller** [comment]
> > Seems like this needs to also capture the new field.
> 
> It's self-explained in the CR [spec](https://github.com/kyverno/kyverno/pull/9157/files#diff-65ed24ceacc89ddea618d14d404077b07db0820a18e49586f8d78a0c27cce09eR102-R107), do we want to call it out in docs?

Yes, spec-level fields are all documented [here](https://kyverno.io/docs/writing-policies/policy-settings/).

**Comment by realshuting** [comment]
> > > Seems like this needs to also capture the new field.
> > 
> > 
> > It's self-explained in the CR [spec](https://github.com/kyverno/kyverno/pull/9157/files#diff-65ed24ceacc89ddea618d14d404077b07db0820a18e49586f8d78a0c27cce09eR102-R107), do we want to call it out in docs?
> 
> Yes, spec-level fields are all documented [here](https://kyverno.io/docs/writing-policies/policy-settings/).

Thanks, updated!

**Review by chipzoller — APPROVED** [review]

**Pull request merged**